lib/rest-builder.js

```diff
@@ -424,8 +401,8 @@ RequestBuilder.prototype.build = function(parameters) {
 
 /**
  * Map the request builder to a function
- * @param {[string]} parameterNames The parameter names that define the order of args. It be an array of strings or multiple string arguments
- * @returns {function} A function to invoke the REST operation
+ * @param {String} parameterNames The parameter names that define the order of args. It be an array of strings or multiple string arguments
```

Comment:
Nope, that is not the correct format. It was failing with that so needed to be changed to just `String`.